publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -32,46 +32,145 @@ guide-category: pipelines
 //
 -->
 
-## Intro
+# Introduction
 
-[Pipelines](https://github.com/tektoncd/pipeline/tree/master/docs#usage) enable a continuous input and continuous delivery (CI/CD) workflow. A set of default tasks and pipelines are provided that can be associated with application stacks.  These pipelines validate the application stack is active, build the application stack, publish the image to a container registry, scan the published image, and then deploy the application to the Kubernetes cluster. You can also create your own tasks and pipelines and customize the pre-built pipelines and tasks. All tasks and pipelines are activated by the product operator.
+[Pipelines](https://github.com/tektoncd/pipeline/tree/master/docs#usage) enable a continuous input and continuous delivery (CI/CD) workflow. A set of default tasks and pipelines are provided that can be associated with application stacks. These pipelines leverage steps and tasks that provide the following capabilities:
+- build the application stack
+- enforce the goverance policy
+- publish the image to a container registry
+- scan the published image
+- sign the image
+- retag an image
+- deploy the application to the Kubernetes cluster
+- promote a service to a gitops repo (This feature is Tech Preview in this release)
+- deploy the application using Kustomize (This feature is Tech Preview in this release)
```

Comment:
instead of "the application" perhaps "a complete microservice scenario" would be better, because a GitOps repo represents the deploy state of more than one microservice.